tests/integration/url_generator_only/test_url_generator_only.py

```diff
@@ -154,3 +155,49 @@ def test_function_imports():
+
+#static dictionary for testing the implementation of exclusion_list
+exclusion_dict = {
+                "$ENTITY_SET$":{
+                        "Products": {
+                            "Properties":["ProductID", "ProductName"],
+                            "Nav_Properties":["Category"]
+                        },
+                        "Categories": {
+                            "Properties":[],
+                            "Nav_Properties":[]
+                        }
+                    }
+                }
+
+def test_method_for_exclusion_dict(method_type):
+    #Method used for testing the implementation of exclusion_list
+    os.environ["ODFUZZ_IGNORE_METADATA_RESTRICTIONS"] = "True"
+    path_to_metadata = Path(__file__).parent.joinpath("metadata-northwind-v2.xml")
+    metadata_file_contents = path_to_metadata.read_bytes()
+    restrictions = RestrictionsGroup(None, exclusion_dict)
+    del_builder = DirectBuilder(metadata_file_contents, restrictions, method_type)
+    del_entities = del_builder.build()
+    queryable_factory = SingleQueryable
+    queries_list = []
+    queries_list.clear()
+    
+    for queryable in del_entities:
+        entityset_urls_count = len(queryable.entity_set.entity_type.proprties())
+
+        for _ in range(entityset_urls_count):
+            q = queryable_factory(queryable, logger, 1)
+            queries,body = q.generate()
+
+            print(method_type, "\n")
+            print(queries.query_string,"\n")
+            print(body,"\n\n")
+ 
+    #FioriDast update the Exception List
+    restrictions = RestrictionsGroup(None, exclusion_dict)
+    del_builder.restrictions = restrictions
```

Comment:
I don't see the exposed property - is this test passing? Travis build was somehow not triggered for this PR.. :(